src/main/java/com/qa/hubspot/page/DealsPage.java

```diff
@@ -0,0 +1,12 @@
+package com.qa.hubspot.page;
+
+import com.qa.hubspot.base.BasePage;
+
+public class DealsPage extends BasePage{
+	
+	public DealsPage() {
+		System.out.println("deals page const...");
```

Comment:
where is the code?
where is page locators?
where is page actions?
Please complete this page class.